packages/client/src/components/app/forms/AttachmentField.svelte

```diff
@@ -78,14 +85,14 @@
   {validation}
   {span}
   {helpText}
-  type="attachment"
+  {type}
   bind:fieldState
   bind:fieldApi
   defaultValue={[]}
```

Comment:
We'll also need to override the defaultValue here as the default value of an array is incorrect for a single attachment field, and causes live eval to show that the value of a new single attachment field is an array, which is a bit confusing for the user.

![image](https://github.com/Budibase/budibase/assets/9075550/a4bf7c19-8404-497b-bf9d-967aa22602d5)

So I think we should override this in the single attachment field to set the default value to null.